src/components/utils/typeCheckers.ts

```diff
@@ -0,0 +1 @@
+export const isFunction = (value: unknown): value is Function => typeof value === 'function';
```

Comment:
there is a lodash in dependencies with same function

Reply:
Yes, but it does a lot of extra job and doesn't help typescript